libs/core/src/lib/badge/badge.component.ts

```diff
@@ -0,0 +1,9 @@
+import { Component, ViewEncapsulation } from '@angular/core';
+
+@Component({
+    selector: 'fd-badge',
+    template: `<div class="fd-badge"><ng-content></ng-content></div>`,
```

Comment:
Maybe let's make it same as button, inline component with `[fd-badge]` selector.

Reply:
So would it be a div or a span? What is the advantage here? Makes sense for an actionable element like button but i don't think so for this one